addons/sys_list/initSettings.sqf

```diff
@@ -0,0 +1,101 @@
+//--------------------------- UI Customization
+// Hint Background Color
+[
+    QGVAR(hintBgColor),
+    "COLOR",
+    localize LSTRING(hintBgColor_displayName),
+    "ACRE2 UI",
+    [ACRE_NOTIFICATION_BG_BLACK],
+    false,
+    {}
+] call CBA_Settings_fnc_init;
+
+// Transmit Color
+[
+    QGVAR(broadcastColor),
+    "COLOR",
+    localize LSTRING(broadcastColor_displayName),
+    "ACRE2 UI",
+    [ACRE_NOTIFICATION_YELLOW],
+    false,
+    {}
+] call CBA_Settings_fnc_init;
+
+// Channel Switch Color
+[
+    QGVAR(switchChannelColor),
+    "COLOR",
+    localize LSTRING(switchChannelColor_displayName),
+    "ACRE2 UI",
+    [ACRE_NOTIFICATION_PURPLE],
+    false,
+    {}
+] call CBA_Settings_fnc_init;
+
+// Toggle Headset Color
+[
+    QGVAR(toggleHeadsetColor),
+    "COLOR",
+    localize LSTRING(toggleHeadsetColor_displayName),
+    "ACRE2 UI",
+    [ACRE_NOTIFICATION_PURPLE],
+    false,
+    {}
+] call CBA_Settings_fnc_init;
+
+// Cycle Radios Color
+[
+    QGVAR(cycleRadiosColor),
+    "COLOR",
+    localize LSTRING(cycleRadiosColor_displayName),
+    "ACRE2 UI",
+    [ACRE_NOTIFICATION_PURPLE],
+    false,
+    {}
+] call CBA_Settings_fnc_init;
+
+// Babel Color
+[
+    QGVAR(languageColor),
+    "COLOR",
+    localize LSTRING(languageColor_displayName),
+    "ACRE2 UI",
+    [ACRE_NOTIFICATION_RED],
+    false,
+    {}
+] call CBA_Settings_fnc_init;
+
+// Hint Text Font
+[
+    QGVAR(hintTxtFont),
+    "LIST",
+    localize LSTRING(hintTxtFont_displayName),
+    "ACRE2 UI",
+    [
```

Comment:
Prettify this, save into array above:
```sqf
private _fonts = ["EtelkaMonospacePro", "EtelkaMonospaceProBold", "LCD14", "PuristaBold", "PuristaLight", "PuristaMedium", "PuristaSemibold", "RobotoCondensed", "RobotoCondensedBold", "RobotoCondensedLight", "TahomaB"];
```
and use:
```sqf
[_fonts, _fonts, 10],
```

Alternatively name the visible names with spaces and no abbreviations and indent properly such as:
```sqf
[
    [
        "Font1"
    ]
]
```
or write them out in one line.

Second is more work for maintenance of course, but looks prettier.

Reply:
This ok for the names portion?
```private _fontNames = ["Etelka Monospace Pro", "Etelka Monospace Pro Bold", "LCD14", "Purista Bold", "Purista Light", "Purista Medium", "Purista Semibold", "Roboto Condensed", "Roboto Condensed Bold", "Roboto Condensed Light", "TahomaB"];```

Reply:
`LCD14` -> `LCD 14`
`Purista Semibold` -> `Purista SemiBold`
`TahombaB` -> `Tahoma B`

Reply:
Committed for additional review